integration-cli/docker_api_containers_test.go

```diff
@@ -1487,11 +1448,7 @@ func (s *DockerAPISuite) TestPostContainersCreateMemorySwappinessHostConfigOmitt
 	containerJSON, err := apiClient.ContainerInspect(testutil.GetContext(c), ctr.ID)
 	assert.NilError(c, err)
 
-	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.31") {
-		assert.Equal(c, *containerJSON.HostConfig.MemorySwappiness, int64(-1))
-	} else {
-		assert.Assert(c, containerJSON.HostConfig.MemorySwappiness == nil)
-	}
+	assert.Assert(c, containerJSON.HostConfig.MemorySwappiness == nil)
```

Comment:
For reviewers; these conditions were changing the test based on version of the daemon (`testEnv.DaemonAPIVersion()` is the fixed API version supported by the daemon), and adjusting the test for bugs in older versions of the daemon (not so much "chances between API versions")